README.md

```diff
@@ -3,6 +3,8 @@
 Plugin for [Yazi](https://github.com/sxyazi/yazi) to preview markdown files with [glow](https://github.com/charmbracelet/glow). To install, clone the repo inside your `~/.config/yazi/plugins/`:
```

Comment:
Remove completely the part related to cloning the repo and leave only the `ya pack` command